### ISSUE TITLE
Battery remaining time gets stuck at 00:00 until bar is restarted

### ISSUE THREAD

**Issue by AppIns** [jaagr/polybar#839]
I noticed that when I suspend my laptop, my battery's remaining life will get stuck at 00:00 (formatted %hour%:%minute%) and will stay that way until the bar is restarted.

**Comment by patrick96** [comment]
What is the output of `acpi` when that happens?

**Comment by AppIns** [comment]
`Battery 0: Discharging, 96%, 24:19:36 remaining`
Also I got a screenshot, sometimes it keeps the percentage, other times it doesn't (this time it went to 0%)
![screenshot from 2017-11-12 14-29-41](https://user-images.githubusercontent.com/13598541/32704192-0c4eb074-c7b6-11e7-86ef-e62fc742c46f.png)

**Comment by EnzDev** [comment]
It happens for me when the computer get out of sleeping.

**Comment by AppIns** [comment]
Yep. If someone wants to reproduce this issue, try plugging in your laptop, sleeping it, unplugging, and then waking it up. If that doesn't work, try mixing up the order. Sometimes all it takes is just sleeping the laptop.

**Comment by patrick96** [comment]
What is the output of the following command, when polybar behaves this way:
```bash
find -L /sys/class/power_supply -maxdepth 2 -type f -print0 2>/dev/null | xargs -0 -I '{}' sh -c 'echo "{}:"; cat {}; echo'
```
This command lists the contents of all the files in the `/sys/class/power_supply` folder.

**Comment by EnzDev** [comment]
Everythin seems normal when it happen. See [Gist](https://gist.github.com/EnzDev/9ebd76d09cde35a5494f10a71c874097).

**Comment by patrick96** [comment]
Yes indeed, it seems everything is in order in those files. What version of polybar are you running? (`polybar -vvv`)
Can you also run polybar with `-l trace`, reproduce this and post the whole output here. If trace logging is not enabled, you'll need to recompile polybar in debug mode (You can do this by adding the `-DCMAKE_BUILD_TYPE=Debug` flag to cmake)

**Comment by EnzDev** [comment]
I'm using the last git tag (3.0.5).
While trying to reproduce the bug with a freshly compiled polybar, I ran into a very similar problem (which I guess has the same origin). 
The battery module just stopped updating (Stucked at `46% (00:57)`) and looped with the debug message
`* module/battery: Polling values (inotify fallback)`
so polybar just stopped rendering (because the config just contains the battery module).
I checked the acpi command and BAT0 files, as before, everything is in order.
Plugging or unplugging the charger didn't do anything but restarting polybar did fix it.
I guess the 00:00 problem is just the same but while polling value, a default 0 popped somewhere.
Here is a chunck of the logtrace :
```
* Redrawing bar window
- renderer: flush(3 geom=84x16+1282+0, falloff=0)
* module/battery: Polling values (inotify fallback)
- module/battery: Inotify event reported for /sys/class/power_supply/BAT0/charge_now
* module/battery: Polling values (inotify fallback)
- module/battery: Inotify event reported for /sys/class/power_supply/BAT0/charge_now
* module/battery: Polling values (inotify fallback)
- module/battery: Inotify event reported for /sys/class/power_supply/BAT0/charge_now
* module/battery: Polling values (inotify fallback)
- module/battery: Inotify event reported for /sys/class/power_supply/BAT0/charge_now
* module/battery: Rebuilding cache => Sleeping the computer here
* Redrawing bar window
- renderer: flush(3 geom=84x16+1282+0, falloff=0)
* module/battery: Polling values (inotify fallback)
* module/battery: Polling values (inotify fallback)
* module/battery: Polling values (inotify fallback)
* module/battery: Polling values (inotify fallback)
(An hundred few more and finaly exiting with ^C) 
warn: Termination signal received, shutting down...
- controller: Uninstall sighandler
- controller: Detach signal receiver
- controller: Stop modules
* module/battery: Stopping
* module/battery: Polling values (inotify fallback)
- module/battery: Release sleep lock
- module/battery: End of subthread
- module/battery: Deconstructing
* Deconstruction of module/battery took 61 ms.
- controller: Joining threads
* Deactivating tray manager
- tray: Unset selection owner
- tray: Unembed clients
- tray: Destroy window
* Waiting for spawned processes to end
* Reached end of application...
```
It seems that gdb is going to be useful.

**Comment by EnzDev** [comment]
Just noticed that this issue is a duplicate of #469

**Comment by patrick96** [comment]
In #469 they get an error, you just get the wrong values. The two may be related but I don't think they're the same bug. I'll need to have look at where the code sets the proper battery values and why they are not set again after the laptop goes to sleep.

**Comment by cvoges12** [comment]
I have the same issue.
